config/samples/appstudio_v1alpha1_dependencyupdatecheck.yaml

```
@@ -7,6 +7,14 @@ metadata:
     app.kubernetes.io/part-of: mintmaker
     app.kubernetes.io/managed-by: kustomize
     app.kubernetes.io/created-by: mintmaker
-  name: dependencyupdatecheck-sample
+  generateName: dependencyupdatecheck-
 spec:
-  # TODO(user): Add fields here
+  workspaces:
+  - workspaceName: "workspace1-tenant"
```

```suggestion
  - workspace: "workspace1"
```
`{workspace}-tenant` is the namespace name in Konflux.